laboratorios/laboratorio-01/2019-1/pregunta-02/main.cpp

```diff
@@ -0,0 +1,195 @@
+#include <iostream>
+#include <iomanip>
+#include <cmath>
+#include <fstream>
+#include <cstring>
+
+void PopulateGameBoard(int t, char **tablero);
+void ShowGameBoard(int t, char **tablero);
+void SearchWord(char *word, int t, char **tablero, char *res);
+bool SearchRight(char *word, int t, char **tablero, char *res);
+bool SearchLeft(char *word, int t, char **tablero, char *res);
+bool SearchDown(char *word, int t, char **tablero, char *res);
+bool SearchUp(char *word, int t, char **tablero, char *res);
+
+
+int main() {
+    std::cout << "Ingresar el tamaño de la cuadricula: \n";
+    int t, cant;
+    std::cin >> t;
+    char **tablero, **word, res[3];
+    // Inicializamos el tablero
+    tablero = new char*[t];
+    for (int i = 0; i < t; i++) {
+        tablero[i] = new char[t];
+    }
+    PopulateGameBoard(t, tablero);
+    ShowGameBoard(t, tablero);
+    std::cout << "Ingrese la cantidad de palabras a buscar\n";
+    std::cin >> cant;
+    word = new char*[t + 1];
+    for (int i = 0; i < t; i++) {
+        word[i] = new char[t + 1];
+    }
+    //Lectura de las palabras
```

Comment:
Try to be more consistent. In the previous comment you have used spacing between the slashes and the text.
```suggestion
    // Lectura de las palabras
```